internal/keepers/ocr.go

```diff
@@ -11,6 +11,14 @@ import (
 	"golang.org/x/crypto/sha3"
 )
 
+const (
+	// observationKeysLimit is the max number of keys that Observation could return.
```

Comment:
nit. we do not send keys anymore in observation

observationUpkeepLimit is the max number of upkeeps that can be sent in an observation

Reply:
Fixed